src/java/org/apache/cassandra/cql3/statements/schema/AlterTypeStatement.java

```diff
@@ -49,10 +49,12 @@
 public abstract class AlterTypeStatement extends AlterSchemaStatement
 {
     protected final String typeName;
+    protected final boolean ifTypeExists;
 
-    public AlterTypeStatement(String keyspaceName, String typeName)
+    public AlterTypeStatement(String keyspaceName, String typeName, boolean ifTypeExists)
     {
         super(keyspaceName);
+        this.ifTypeExists = ifTypeExists;
```

Comment:
ifTypeExists  -> ifExists

Reply:
makes sense. Can be derived from the context/classname and explicitly naming the variable is redundant.